pkg/announce/github_page.go

```diff
@@ -109,6 +110,7 @@ type SBOMOptions struct {
 	Repo          string
 	RepoDirectory string
 	Tag           string // Version Tag
+	Format        string // "tag-value"  | "json"
```

Comment:
Nit: make it an enum.

Reply:
I mean making the options constants, like there:

https://github.com/kubernetes-sigs/security-profiles-operator/blob/b619a8f8ba2e5a9f6d502bced6f99d6a2bb1b7f3/internal/pkg/cli/recorder/consts.go#L37-L48